doc/4-instrument/es5503.md

```diff
@@ -0,0 +1,45 @@
+# ES5503 instrument editor
+
+ES5503 instrument editor is divided into 4 tabs:
+
+- **ES5503**: for controlling the parameters of ES5503 sound channel.
+- **Wavetable**: for wavetable-related controls.
+- **Macros**: for macros controlling volume/arp/pitch/panning/etc.
+- **Sample**: for sample-related controls.
+
+## ES5503
+
+There is a tricky feature of ES5503 when it comes to playing wavetables/samples. ES5503 can play waves of sizes 256, 512, ..., 32768 steps. When it encounters `0` in sample memory it stops playing the wavetable/sample, no matter what oscillator mode and wave size are.
```

Comment:
Perhaps "waves of sizes that are multiples of 256"? Either that or add "768" to the sequence to make it clear.

Reply:
I should have continued... It's 256, 512, 1024, 2048, 4096, 8192, 16384 and 32768